src/realtime_metrics/main.py

```diff
@@ -395,6 +412,60 @@ def get_next_actual_arrival(timestamp: int, route_id: str, stop_id: str) -> Stop
     return actual_arrivals[0]
 
 
+def prediction_inconsistency(actual_arrival_time: int, updates: list[tuple[TripUpdate, StopTimeUpdate]]) -> float:
+    """
+    Computes the prediction inconsistency metrics for a route, trip and stop combination.
+    The metric is defined here: https://docs.google.com/document/d/1-AOtPaEViMcY6B5uTAYj7oVkwry3LfAQJg3ihSRTVoU. 
+    It computes the predicted arrival time spread in 30 two minute time frames, starting every minute.
+    It computes the spread as the difference of the minimal and maximal predicted arrival time in each time frame. 
+    The prediction inconsistency is then the average spread of all windows.
+
+    Parameters:
+    actual_arrival_time: the actual arrival time of the vehicle (for that route, trip and stop)
+    updates: list of corresponding trip updates and stop time updates (for that route, trip and stop)
```

Comment:
Same here and anywhere else if I didn't catch it
```suggestion
    updates: list of corresponding trip updates and stop time updates (for that trip and stop)
```